packages/web3torrent/scripts/start.js

```diff
@@ -1,23 +1,21 @@
 const fs = require('fs');
 
-// TODO connect to *existing* ganache. Here we start a new instance to aid development.
-
 // Do this as the first thing so that any code reading it knows the right env.
 process.env.BABEL_ENV = 'development';
 process.env.NODE_ENV = 'development';
 
-let ganacheServer;
 let devServer;
+let trackerServer;
 
 // Makes the script crash on unhandled rejections instead of silently
 // ignoring them. In the future, promise rejections that are not handled will
 // terminate the Node.js process with a non-zero exit code.
 process.on('unhandledRejection', err => {
-  if (ganacheServer) {
-    ganacheServer.close();
-  }
   if (devServer) {
-    devServer.close();
+    devServer.kill();
```

Comment:
`devServer` does not have a `.close()` method, since it is a `ChildProcess`. We can `kill()` it, but its children might not be killed.